src/nvim/lua/xdiff.h

```diff
@@ -0,0 +1,10 @@
+#ifndef NVIM_LUA_XDIFF_H
+#define NVIM_LUA_XDIFF_H
+
+#include <lua.h>
+#include <lualib.h>
+#include <lauxlib.h>
+
+int nlua_xdl_diff(lua_State *lstate);
```

Comment:
use a `INCLUDE_GENERATED_DECLARATIONS` block instead. look at `lua/treesitter.h` (and similarly for the .c file).

Reply:
Done